portality/templates/head.html

```diff
@@ -7,9 +7,14 @@
     <meta name="author" content="{{ app.config['ADMIN_NAME'] }}">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-
     <link href="{{ url_for('static', filename='doaj/images/favicon.ico') }}" rel="icon" type="image/x-icon" />
 
+    <!-- Open Graph tags for Facebook -->
+    <meta property="og:title" content="{{ app.config['SERVICE_NAME'] }}"/>
+    <meta property="og:url" content="{{ app.config['BASE_URL'] }}" />
```

Comment:
Should this always be the base then? Like if I load the search page (since head.html gets included in every single page load), this should still be https://doaj.org, not https://doaj.org/search ?